examples/in_toto_run_test.rs

```diff
@@ -8,7 +8,7 @@ fn main() {
 
     let link = in_toto_run(
         "example",
-        Some("tests"),
+        Some("tests/test_runlib"),
```

Comment:
I am not getting a different materials or products list changing the runlib.

Reply:
Huh, I can't reproduce what I thought I had seen. Must have just missed it. Looks fine with the following command:

    RUSTFLAGS=-Awarnings cargo run --example in_toto_run_test --quiet | sed 's/Generated link: //' | jq .